### PR TITLE
Team 1 Diamond Kata

### PR DESCRIPTION
**Pull request by SebastianHerzog**

### CODE REVIEW

File: src/main/java/de/cas/mse/exercise/diamond/Diamond.java [hunk 1]
```diff
@@ -4,26 +4,26 @@
 
 public class Diamond {
 
-	public String print(int n) {
```
Comment (DragonflyCode11) [inline src/main/java/de/cas/mse/exercise/diamond/Diamond.java:7]: Print is Wrong: Creates a String and does not print it.

File: src/main/java/de/cas/mse/exercise/diamond/Diamond.java [hunk 2]
```diff
@@ -4,26 +4,26 @@
...
 			builder.insert(0, chars);
 			builder.append(chars);
 		}
 		return builder.toString();
 	}
 
-	private char[] make(int i, int j) {
-		int amount = ((i - j) / 2);
-		char[] chars = new char[amount + j + 1];
+	private char[] createLayer(int height, int width) {
```
Comment (DragonflyCode11) [inline src/main/java/de/cas/mse/exercise/diamond/Diamond.java:20]: What is a layer in this context? Line/Row would be better

Comment (Bananamafia) [inline src/main/java/de/cas/mse/exercise/diamond/Diamond.java:20]: height und width ist gut, aber vielleicht totalHeight?

File: src/main/java/de/cas/mse/exercise/diamond/Diamond.java [hunk 3]
```diff
@@ -4,26 +4,26 @@
...
+		int amount = ((height - width) / 2);
```
Comment (Bananamafia) [inline src/main/java/de/cas/mse/exercise/diamond/Diamond.java:21]: ggf. amount of what

File: src/main/java/de/cas/mse/exercise/diamond/Diamond.java [hunk 4]
```diff
@@ -4,26 +4,26 @@
...
+		char[] chars = new char[amount + width + 1];
```
Comment (DragonflyCode11) [inline src/main/java/de/cas/mse/exercise/diamond/Diamond.java:22]: Line/Row would fit better. Or in this case. Layer since this method is called layer.

Comment (Bananamafia) [inline src/main/java/de/cas/mse/exercise/diamond/Diamond.java:22]: ggf. chars of what

**Pull request merged**